pallets/staking/src/mock.rs

```diff
@@ -397,7 +397,7 @@ impl pallet_parameters::Config for Test {
 pub fn new_test_ext() -> sp_io::TestExternalities {
     let mut t = system::GenesisConfig::<Test>::default().build_storage().unwrap();
     let pallet_balances = pallet_balances::GenesisConfig::<Test> {
-        balances: vec![(1, 100), (2, 100), (3, 100), (4, 100)],
+        balances: vec![(1, 100), (2, 100), (3, 100), (4, 100), (7, 100), (8, 100), (101, 200)],
```

Comment:
```suggestion
        balances: vec![(1, 100), (2, 100), (3, 100), (4, 100), (7, 100), (8, 100), (9, 100)],
```

For consistency? Or maybe can you explain why you had it as `(101, 200)`?

Reply:
nah I just needed a number I knew was not used, 9 probably works